application/src/main/java/org/togetherjava/tjbot/commands/mathcommands/wolframalpha/WolframAlphaImages.java

```diff
@@ -35,7 +34,10 @@ enum WolframAlphaImages {
             new FontRenderContext(new AffineTransform(), true, true);
     private static final Color TITLE_COLOR = Color.decode("#3C3C3C");
     private static final Font TITLE_FONT = new Font("Arial", Font.BOLD, 15);
-    private static final int TITLE_HEIGHT_PX = 20;
```

Comment:
Why did you kill that variable?

Reply:
> Why did you kill that variable?

sonar lint told me to do so

Reply:
![image](https://user-images.githubusercontent.com/73871477/186442616-e0c50ea6-9da1-42e0-b5d3-8235bdd33207.png)

Reply:
small note: u also gotta check whether its truly meant to be unused, i.e. maybe someone in the code just wrote `20` somewhere instead of using the constant. 

did that for u though. seems that its truly unused and also not needed anymore. alright 👍